autoortho/aoconfig.py

```diff
@@ -45,6 +45,43 @@ def __eq__(self, other):
         return NotImplemented
 
 
+def clean_cache(cache_dir, size_gb):
+
+    log.info(f"Cleaning up cache_dir {cache_dir}")
+
+    target_gb = max(size_gb, 10)
```

Comment:
I would not limit that in this function. It might be useful to keep that more generic.
```suggestion
    target_gb = size_gb
```

Reply:
For reasons stated above, not going to support auto clear to 0.